internal/app/wwctl/container/copy/root.go

```diff
@@ -0,0 +1,33 @@
+package copy
+
+import (
+	"github.com/hpcng/warewulf/internal/pkg/container"
+	"github.com/spf13/cobra"
+)
+
+var (
+	baseCmd = &cobra.Command{
+		DisableFlagsInUseLine: true,
+		Use:                   "cp CONTAINER NEW_NAME",
```

Comment:
please rename to
```
Use: "copy CONTAINER NEW_NAME",
Aliases: []string{"cp"},
```
as all the container have long name for the commands, e.g. `delete` and not `rm` having `cp` as alias is a good alternative here.

Reply:
Sure !